### DISCUSSION TITLE
Forward Proxy server support

### DISCUSSION THREAD

**Discussion by mikocot** [jgdevlabs/aspnetcore-recaptcha#18] [General]
Hi, it's not uncommon for the servers to make use of proxys to route outcoming requests, like the one needed for recaptcha. This is also the case for our company.

I can think of a workaround for now, but in the long term it would be nice to add this feature, especially, it is quite trivial.
We'd only need to allow configuring additional values and then include them when creating an http client.
I'll submit a PR shortly.

**Comment by jooni91** [comment]
This feature just went live on NuGet.

**Reply by mikocot** [reply]
Thanks, I've replaced our custom code with the new version earlier this week and it seems to work pefectly fine :)
